### ISSUE TITLE
Zooming with the mouse wheel is jumpy

### ISSUE THREAD

**Issue by tfili** [AnalyticalGraphicsInc/cesium#4537]
When you use the right mouse button to zoom it is very smooth, but when you use the wheel it is very jumpy. For the sake of consistency we should have the wheel move smoothly as well.

**Comment by emackey** [comment]
@bagnell could this be done by having the wheel impart zoom-inertia, rather than simply change the position?

**Comment by bagnell** [comment]
Yes. The wheel used to have inertia. @hpinkos tracked it all the way back to b24 when it was removed.

**Comment by pjcozzi** [comment]
CC #2968

**Comment by AliFlux** [comment]
Any idea if this feature is coming back? Google Earth zoom is pretty smooth, and there's another opensource lib called OpenGlobus with same same inertial zoom:
http://openglobus.org/examples/emptyTerrain/emptyTerrain.html

**Comment by OmarShehata** [comment]
@AliFlux if you'd like to propose an implementation of this in a PR that could help generate interest/see if others would like to see this change too.
